.github/workflows/recommend-integration-tests.yml

```diff
@@ -29,46 +29,43 @@ jobs:
       - name: Has diff?
         run: echo ${{ steps.source-files.outputs.diff != '' }}
 
-      - name: Get or Create Comment
+      - name: Add label and comment
```

Comment:
Note for reviewer: Merged the comment and label tasks because there is a lot of overlap now

Reply:
Thanks for the detailed explanation!